util/typeutil/stringInt_test.go

```diff
@@ -0,0 +1,28 @@
+package typeutil
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUnmarshalJSON(t *testing.T) {
```

Comment:
Since test are package scoped, please include the type name, ex: `TestStringIntUnmarshalJSON` or since there is no other functionality for the StringInt type, `TestStringInt` would also be sufficient.

Reply:
this is done - https://github.com/prebid/prebid-server/pull/2617/commits/0141e91006f95b9a243dc679fd28b050750951bf